app/protocols/browser-protocol.js

```diff
@@ -24,7 +24,7 @@ module.exports = async function createHandler () {
     const { pathname, hostname } = parsed
     const toResolve = path.join(hostname, pathname)
 
-    if (toResolve === 'theme/vars.css') {
+    if ((hostname === 'theme') && (pathname === '/vars.css')) {
```

Comment:
I think the latest `master` already has a fix for this, so it might be good to revert it

Reply:
Strange... Why is this comparing with an old `master`?

Reply:
Uhh, I think I just confused myself. :P NVM looks good